src/tmplits/tmplits-tabs/library.handlebars

```diff
@@ -0,0 +1,47 @@
+<script id="TabSample" type="text/x-handlebars-template">
+    <lui-tab-control value="1">
+        <template title="Page1">
+            <h1>Page 1</h1>
+            {{#tmplit 'Columns' }}
+                {{tmplit "Numeric" 'Test' type='input' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+                {{tmplit 'Numeric' 'label' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+                {{tmplit 'Numeric' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+            {{/tmplit}}
+        </template>
+        <template title="Page2">
+            <h1>Page 2</h1>
+            {{#tmplit 'Columns' }}
+                {{tmplit 'Numeric' 'Test' type='input' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+                {{tmplit 'Numeric' 'label' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+                {{tmplit 'Numeric' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+            {{/tmplit}}
+        </template>
+    </lui-tab-control>
+
+    {{#tmplit 'Tabs' id="tabs4" value='2'}}
+        <template title="Page1">
```

Comment:
Could these be a tab item tmplit?

Reply:
i suppose there could be a

{{tmplit 'TabItem'}}

Is that the question?

Reply:
yup, nice i see you added that